### DISCUSSION TITLE
Pro Tools Silicon Beta.

### DISCUSSION THREAD

**Discussion by 100below** [apohl79/audiogridder#1125] [Ideas]
Running Pro Tools 22.12beta (arm) with AudioGridderPlugin_1.2.0-beta15_macOS-arm64. So far it is working well although I have not stress-tested it yet.

The newly released beta of Pro Tools 22.12 is universal binary, allowing manual switching between Intel and arm64. It would be great if the AudioGridder AAX plugin was a self-switching universal binary, the way most of the new AAX plugins are designed. Many of us will be going back and forth between the two Pro Tools versions for the foreseeable future. The current requirement to reinstall the AudioGridder plugin to switch between versions is doable but a little awkward.

Great work having this work on launch!

**Comment by apohl79** [comment]
Thats great! Thx for the news. Maybe the universal build of AG supports that.

**Comment by 100below** [comment]
I missed that the universal builds were available! 
I switched to the universal builds, and again the basics seem to be working well, but I have a question. I unchecked the server to load it as ARM, and it works within Protools / ARM. What I don't understand is that it loads non-silicon plugins in arm mode. Most apps crash or grind to a halt mixing the two. Any idea what's happening?
